QuickBooks/IPP.php

```diff
@@ -961,6 +963,22 @@ public function version($version = null)
 		
 		return $this->_ids_version;
 	}
+
+	/**
+	 * Get or set the sandbox mode. If not set, will default false for production
+	 *
+	 * @param bool $sandbox		Set as true for sandbox mode or false for production
+	 * @return bool				Returns true if in sandbox mode. False for production mode
+	 */
+	public function sandbox($sandbox = null)
+	{
+		if ($sandbox)
```

Comment:
You should do `isset($sandbox)` instead because currently if you turn on sandbox mode it's impossible to turn it off.